### PR TITLE
A minor resprite, turned cargo gun rework, along with new weapons

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

So, basicly what this does is resprites the AKs and fixes the overlays, but also adds a disposable gun, which is not only printable if you have the disk, but buyable from cargo
this also increases most guns cost by 3x

## Why It's Good For The Game

resprites are nice, balances are also nice

## Changelog
:cl:
add: Adds a new, disposable gun
imageadd: new sprites by @any%
balance: Most guns in cargo are now worth x3
/:cl:

### CODE REVIEW

**Comment by WindowsErrors** [comment]
ammunition in the biden economy

**Comment by zestybastard** [comment]
> 
> 
> ammunition in the biden economy

at least you always have your Papa John's Cheasy Breasy Extra Pepperoni Liberator

**Comment by Valorium-RB** [comment]
Guns and money is hard enough to get as-is, tripling the cost of half-decent firearms just gives ships with actual round start firearms a massive advantage over normal ones. If we're tripling the cost of firearms, every ship needs multiple firearms with spare ammunition from the word go.

**Comment by Sun-Soaked** [comment]
> Guns and money is hard enough to get as-is, tripling the cost of half-decent firearms just gives ships with actual round start firearms a massive advantage over normal ones. If we're tripling the cost of firearms, every ship needs multiple firearms with spare ammunition from the word go.

my response to this is: Ruin/lavaland loot.
(I do think that 3x is a little harsh, as it practically means that most guns are unreasonable to even seek compared to improv weapons and loot)

**Comment by WindowsErrors** [comment]
personally i think instead of making guns as rare as ammunition is
or relying on _miners_ for the security of the ship
my face when shaft miner _(´ཀ`」 ∠)_

i instead propose that to counterbalance the increased rarity of guns, ammunition be made more common by removing that thing where you need to emag the autolathe in order to make ammunition, or whatever else you have in mind

ammunition is consumable but guns are not
this is countered by the fact that ammunition is very easy to make while guns are not as easy to make

a wise man once told me that where possible one must have gear in game better in some ways and worse in other ways compared to existing ones as opposed to better or worse in every single way

**Comment by PrefabQuasar** [comment]
i think the current long term plan for gun balance is to require that people craft their own ammo, once that happens, it would be reasonable for guns to be either relatively cheap, or more prevalent in ruins, but until then, a cost increase seems fine to me, maybe not *as* drastic as x3 though

**Comment by Sun-Soaked** [comment]
> personally i think instead of making guns as rare as ammunition is or relying on _miners_ for the security of the ship my face when shaft miner _(´ཀ`」 ∠)

I think it's important to note that the whole conundrum of "op miner loot vs. station balance" no longer exists because the concept of job exclusivity is dead, and will only decay from here on out. You're completely free as your ship's assistant, medtech, captain, etc. to pick up a Crusher and go style on some watcher tendrils.

In fact, with the places me and others are currently taking mining balance, it'll likely become necessary to have multiple crewmembers working together simultaneously to get said loot safely, making "expeditions" to tendrils, ruins, etc. and such a shipwide project

**Comment by TheNeoGamer42** [comment]
yeah longer term I am pretty sure guns will be more of a ruin thing, and currently the cargo price is being increased because right now cargo is weird and lame and we need outposts and maybe when outposts are a thing we can have cheaper guns, but obviously you need someone selling said guns

**Comment by WindowsErrors** [comment]
the point still stands that it's probably not very satisfying to go through all these hoops to get a ranged weapon and then be given blue balls because you can't even use it without the boolets

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -660,10 +663,46 @@
...
+	desc = "Contains two cheaply made reproductions of the AK-47 by Nanotrasen, the NT-AK."
+	cost = 12000
+	contains = list(/obj/item/gun/ballistic/automatic/ak47/nt,
+					/obj/item/gun/ballistic/automatic/ak47/nt)
+	crate_name = "rifle crate"
+	dangerous = TRUE
+
+/datum/supply_pack/security/armory/akntammo
+	name = "Nanotrasen Brand Kalashnikov Ammo Crate"
+	desc = "Contains 2 30 round properitary magazines for the NT-AK."
```
Comment (MarkSuckerberg) [inline code/modules/cargo/packs.dm]:
```suggestion
	desc = "Contains two 30 round proprietary magazines for the NT-AK."
```

File: whitesands/code/modules/projectiles/guns/ballistic/automatic.dm [hunk 2]
```diff
@@ -94,26 +94,75 @@
...
+	righthand_file = 'whitesands/icons/mob/inhands/weapons/64x_guns_right.dmi'
+	fire_rate = 5
 	mag_display = TRUE
 	weapon_weight = WEAPON_MEDIUM
 	w_class = WEIGHT_CLASS_BULKY
 	mag_type = /obj/item/ammo_box/magazine/ak47
 
+/obj/item/gun/ballistic/automatic/ak47/nt
+	name = "NT-AK"
+	desc = "A cheap rip-off of a already cheap rifle. Comes with a foldable stock for easy storage, although accuracy is questionable when folded. Control click to toggle the stock."
```
Comment (MarkSuckerberg) [inline whitesands/code/modules/projectiles/guns/ballistic/automatic.dm]:
```suggestion
	desc = "A cheap rip-off of an already cheap rifle. Comes with a foldable stock for easy storage, although accuracy is questionable when folded. Control click to toggle the stock."
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
All spelling/grammar fixes, code looks good

**Comment by Cenrus** [comment]
Kalash ammo crate comes with the NT AK ammo

**Comment by rye-rice** [comment]
fuck

File: whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 3]
```diff
@@ -13,11 +13,37 @@
...
 	icon = 'whitesands/icons/obj/ammo.dmi'
-	icon_state = "aks74u_mag"
+	icon_state = "ak47_mag"
 	ammo_type = /obj/item/ammo_casing/ballistic/a545_39
 	caliber = "5.45x39mm"
 	max_ammo = 30
 
+/obj/item/ammo_box/magazine/aks74u/update_icon()
+	..()
+	var/hasammo
```
Comment (ZephyrTFA) [inline whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm:23]: Just assign this using a ternary operator

File: whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 4]
```diff
@@ -13,11 +13,37 @@
...
+	name = "NT AK Magazine (4.6x30mm))"
+	icon = 'whitesands/icons/obj/ammo.dmi'
+	icon_state = "ak47_mag"
+	ammo_type = /obj/item/ammo_casing/c46x30mm
+	caliber = "4.6x30mm"
+	max_ammo = 30
+
+/obj/item/ammo_box/magazine/aknt/update_icon()
+	..()
+	var/hasammo
```
Comment (ZephyrTFA) [inline whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm:40]: See above

File: whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 5]
```diff
@@ -26,6 +52,15 @@
 	caliber = "7.62x39mm FMJ"
 	max_ammo = 30
 
+/obj/item/ammo_box/magazine/ak47/update_icon()
+	..()
+	var/hasammo
```
Comment (ZephyrTFA) [inline whitesands/code/modules/projectiles/boxes_magazines/external/rifle.dm:57]: See above

File: whitesands/code/modules/projectiles/guns/ballistic/automatic.dm [hunk 6]
```diff
@@ -94,26 +94,78 @@
...
+	playsound(src.loc, 'sound/weapons/empty.ogg', 100, 1)
+	update_overlays()
+
+/obj/item/gun/ballistic/automatic/ak47/nt/update_overlays()
+	. = ..()
+	var/mutable_appearance/stock
+	if(!folded)
+		stock = mutable_appearance(icon, "ak47_nt_stock")
+	else
+		stock = mutable_appearance(icon, null)
```
Comment (ZephyrTFA) [inline whitesands/code/modules/projectiles/guns/ballistic/automatic.dm:165]: Null icon state?

Comment (rye-rice) [reply whitesands/code/modules/projectiles/guns/ballistic/automatic.dm:165]: Its supposed to hide the stock when folding it, this was the only way i could think of

**Comment by ValoTheValo** [comment]
can you add the ability to buy stechkin and stechkin ammo?

**Comment by HighPrincessErinys** [comment]
> can you add the ability to buy stechkin and stechkin ammo?

Honestly? This isn't even a terrible idea, now that Stechkins and the like aren't just "antagonist gear! Ooh, aah!"
More support for faction weapons like letting you make Stechies and their ammo with research like you can Commanders, or the same with SolGov's M9Cs, would be pretty cool.

**Comment by ValoTheValo** [comment]
> > can you add the ability to buy stechkin and stechkin ammo?
> 
> Honestly? This isn't even a terrible idea, now that Stechkins and the like aren't just "antagonist gear! Ooh, aah!" More support for faction weapons like letting you make Stechies and their ammo with research like you can Commanders, or the same with SolGov's M9Cs, would be pretty cool.

yeah. its always annoyed me how my syndicate ship needs to use nanotrasen weapons because i cant make more weapons or ammo unless i somehow get an uplink

**Comment by ZephyrTFA** [comment]
You have two conflicts in your dmi

**Comment by rye-rice** [comment]
why are my checks failing i didn't even touch the maps

**Pull request merged**